themes/_core/js/cart.js

```diff
@@ -4,13 +4,13 @@ import prestashop from 'prestashop';
 $(document).ready(() => {
   prestashop.on('cart updated', function (event) {
     var refreshURL = $('.-js-cart').data('refresh-url');
-    var requestData = {};
+    var requestData = {
+      action: 'refresh'
```

Comment:
There already is the `action=refresh` parameter in the `$('.-js-cart').data('refresh-url')`. Could you remove it from the template if you add it in JS please ?